content/blog/pointers-and-references-cpp/index.md

```diff
@@ -0,0 +1,153 @@
+﻿---
+{
+    title: 'Pointers and References in C/C++',
+    description: 'An overview of how pointers and references function in C/C++',
+    published: '2020-05-20T09:40:00.000Z',
+    edited: '2020-05-20T09:40:00.000Z',
+    authors: ['seanmiller'],
+    tags: ['computer science', 'cpp'],
+    attached: [],
+    license: 'cc-by-nc-nd-4'
+}
+---
+
+# Pointers and References
```

Comment:
A lot of the articles on UU from what I can tell don't use an initial heading since the page title kind of steps in for it:
![image](https://user-images.githubusercontent.com/13894696/82517047-19ce1500-9ad1-11ea-89d5-26b58a0df996.png)
![image](https://user-images.githubusercontent.com/13894696/82517075-2a7e8b00-9ad1-11ea-8986-7fa40c63dfde.png)
With an article overview that first heading can help, but on this article it would be

# Pointers and References in C/C++
publishing meta
# Pointers and References

so I'm putting this note here. It might be better to remove this heading and de-nest the other headings a level

Reply:
Agree with this, worth a quick fix